### PR TITLE
add note about setting device count

### LINKED ISSUE #280: Add notes on device count setting in tutorials
> LGTM. Maybe just add a note for the user about the `set_host_device_count` calls and how they may wish to reconfigure on their own systems.

_Originally posted by @dylanhmorris in https://github.com/CDCgov/multisignal-epi-inference/pull/278#pullrequestreview-2186548934_

### PR DESCRIPTION
**Pull request by sbidari**
closes #280

### CODE REVIEW

**Review by damonbayer — CHANGES_REQUESTED** [review]
Thanks @sbidari! Please make the link attached to `set_host_device_count`, rather than "numpyro's" and add also add this note to the hospital_admissions_model tutorial.

**Comment by sbidari** [comment]
Thanks, will make those changes! Any idea why this is failing a test? @damonbayer

**Comment by damonbayer** [comment]
I think we had some automatic merge issue. Looks like @AFg6K7h4fhy2 has implemented a fix [here](https://github.com/CDCgov/multisignal-epi-inference/pull/300). You could make that fix a separate PR or decide some other way to handle it with @AFg6K7h4fhy2.

**Comment by damonbayer** [comment]
> I think we had some automatic merge issue. Looks like @AFg6K7h4fhy2 has implemented a fix [here](https://github.com/CDCgov/multisignal-epi-inference/pull/300). You could make that fix a separate PR or decide some other way to handle it with @AFg6K7h4fhy2.

Resolved in https://github.com/CDCgov/multisignal-epi-inference/pull/327

File: docs/source/tutorials/basic_renewal_model.qmd [hunk 1]
```diff
@@ -33,7 +33,7 @@ from pyrenew.metaclass import (
 import pyrenew.transformation as t
 from numpyro.infer.reparam import LocScaleReparam
 ```
-By default, XLA (which is used by JAX for compilation) considers all CPU cores as one device. Depending on your system's configuration, we recommend using [numpyro's](https://num.pyro.ai/en/stable/utilities.html#set-host-device-count) `set_host_device_count` to set the number of device available for parallel computing. Here, we set the device count to be 2.
+By default, XLA (which is used by JAX for compilation) considers all CPU cores as one device. Depending on your system's configuration, we recommend using numpyro's [set_host_device_count](https://num.pyro.ai/en/stable/utilities.html#set-host-device-count) function to set the number of device available for parallel computing. Here, we set the device count to be 2.
```
Comment (dylanhmorris) [inline docs/source/tutorials/basic_renewal_model.qmd:36]: Typo and formatting.

```suggestion
By default, XLA (which is used by JAX for compilation) considers all CPU cores as one device. Depending on your system's configuration, we recommend using numpyro's [`set_host_device_count()`](https://num.pyro.ai/en/stable/utilities.html#set-host-device-count) function to set the number of devices available for parallel computing. Here, we set the device count to be 2.
```

File: docs/source/tutorials/hospital_admissions_model.qmd [hunk 2]
```diff
@@ -4,16 +4,16 @@ format: gfm
 engine: jupyter
 ---
 
+This document illustrates how a hospital admissions-only model can be fitted using data from the Pyrenew package, particularly the wastewater dataset. The CFA wastewater team created this dataset, which contains simulated data. 
+
+We begin by loading `numpyro` and configuring the device count to 2 to enable running MCMC chains in parallel. By default, XLA (which is used by JAX for compilation) considers all CPU cores as one device. We use numpyro's [set_host_device_count](https://num.pyro.ai/en/stable/utilities.html#set-host-device-count) function to specify the number of device available.
```
Comment (dylanhmorris) [inline docs/source/tutorials/hospital_admissions_model.qmd]: Typo and formatting 
```suggestion
We begin by loading `numpyro` and setting the device count to 2 to enable running MCMC chains in parallel. By default, XLA (which is used by JAX for compilation) considers all CPU cores as one device. We use numpyro's [`set_host_device_count`](https://num.pyro.ai/en/stable/utilities.html#set-host-device-count) function to specify the number of devices available. 
```

**Review by dylanhmorris — CHANGES_REQUESTED** [review]
A couple very minor fixes and then should be ready @sbidari

File: docs/source/tutorials/hospital_admissions_model.qmd [hunk 3]
```diff
@@ -4,16 +4,16 @@ format: gfm
 engine: jupyter
 ---
 
+This document illustrates how a hospital admissions-only model can be fitted using data from the Pyrenew package, particularly the wastewater dataset. The CFA wastewater team created this dataset, which contains simulated data.
+
+We begin by loading `numpyro` and configuring the device count to 2 to enable running MCMC chains in parallel. By default, XLA (which is used by JAX for compilation) considers all CPU cores as one device. We use numpyro's [set_host_device_count](https://num.pyro.ai/en/stable/utilities.html#set-host-device-count) function to specify the number of device available.
```
Comment (damonbayer) [inline docs/source/tutorials/hospital_admissions_model.qmd:9]:
```suggestion
We begin by loading `numpyro` and configuring the device count to 2 to enable running MCMC chains in parallel. By default, XLA (which is used by JAX for compilation) considers all CPU cores as one device. Depending on your system's configuration, we recommend using numpyro's [`set_host_device_count()`](https://num.pyro.ai/en/stable/utilities.html#set-host-device-count) function to set the number of devices available for parallel computing.
```

**Review by dylanhmorris — APPROVED** [review]
LGTM, thanks @sbidari!

**Pull request merged**